### ISSUE TITLE
implement plugin for some file manager on linux

### ISSUE THREAD

**Issue by cryptopuzzle** [ryran/pyrite#27]
It will be very useful and easier to have it as shortcut in nautilus or nemo file manager on linux.
That way users will only right-click on file and select pyrite -> symmetric -> aes256, and then enter password. Not to mention that is option that almost no other program provides (Aescrypt maybe.).

**Comment by ryran** [comment]
Agreed. That would be awesome. I don't have any time to devote to this project. Feel free to fork and work on it.
